src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1512,19 +1512,25 @@ public void TestTimesStringV21()
         [Test]
         public void TestAppend()
         {
-            Hash assigns = Hash.FromAnonymousObject(new { a = "bc", b = "d" });
+            Hash assigns = Hash.FromAnonymousObject(new { a = "bc", b = "d", d = "x" });

```

Comment:
Not clear where d is used below. Seems like this and next are unnecessary changes and just the last two lines are valid.

Reply:
d was added to confirm that the parser didn't get confused between `'d'` and `d` in a template. Again, not necessary, but I thought it a useful.

Reply:
Good thinking. Leave it and comment so nobody removes it later.